### PR TITLE
update lens creation in Readme

### PR DESCRIPTION
**Pull request by julien-truffaut**
@refried what do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,25 +115,22 @@ There are 3 ways to create `Lens`, each with their pro and cons:
...
-     Adding `@Lenses` annotation on case classes will generate `Lens` for every single accessor of the case class.
-     These generated `Lens` are in the companion object of the case class (even if there is no companion object declared).
-     Nevertheless, this solution has several disadvantages: 
+3.   Finally, the fully automatic method using `@Lenses` macro annotation. This solution is the most boiler plate
+     free solution but it has several disadvantages:
      1.   users need to add the macro paradise plugin to their project.
-     2.   IDE have a poor support for Macro annotation, so it is likely your IDE will not know about the generated `Lens` (but it will compile). If you want a better IDE support, please vote on the following [issue](http://youtrack.jetbrains.com/issue/SCL-7419). 
-     3.   this solution can only be applied when you control the case classes since you need to annotate them. This means you cannot use this technique for classes defined in another project.
+     2.   poor IDE supports, at the moment only IntelliJ recognises it.
+     3.   requires to have access to the case classes since you need to annotate it.
```
Comment (NightRa) [inline README.md]: annotate *them*

File: README.md [hunk 2]
```diff
@@ -115,25 +115,22 @@ There are 3 ways to create `Lens`, each with their pro and cons:
      val _company = Lens((_: Employee).company)( c => e => e.copy(company = c))
      ```
 
-2.   We can use the `Lenser` macro to create a sort of `Lens` factory. This solution is limited to case classes:
+2.   The semi-automatic method using `Lenser` blackbox macro:
```
Comment (NightRa) [inline README.md]: using **the** `Lenser` blackbox macro

File: README.md [hunk 3]
```diff
@@ -115,25 +115,22 @@ There are 3 ways to create `Lens`, each with their pro and cons:
...
      
-     // or in a single line
+     // or
+     val lenser = Lenser[Employee]
      val (_company, _name) = (lenser(_.company) , lenser(_.name))
      ```
 
-3.   Finally, the boiler plate free solution with macro annotation (which are probably the most experimental part of macros).
-     Adding `@Lenses` annotation on case classes will generate `Lens` for every single accessor of the case class.
-     These generated `Lens` are in the companion object of the case class (even if there is no companion object declared).
```
Comment (NightRa) [inline README.md]: These two lines are critical.

File: README.md (hunk 2 above)
Comment (julien-truffaut) [inline README.md]: are you sure? It sounds better without *the* to me

File: README.md (hunk 3 above)
Comment (julien-truffaut) [inline README.md]: true

File: README.md (hunk 2 above)
Comment (refried) [inline README.md]: +the :-)

File: README.md [hunk 4]
```diff
@@ -115,40 +115,40 @@ There are 3 ways to create `Lens`, each with their pro and cons:
...
-     Nevertheless, this solution has several disadvantages: 
+3.   Finally, the fully automatic method using `@Lenses` macro annotation.
+     `@Lenses` generate `Lens` for every accessor of a case class in its companion object (even if there is no companion object defined).
```
Comment (refried) [inline README.md]: @Lenses generates a Lens for...

File: README.md [hunk 5]
```diff
@@ -115,40 +115,40 @@ There are 3 ways to create `Lens`, each with their pro and cons:
...
+     This solution is the most boiler plate free but it has several disadvantages:
      1.   users need to add the macro paradise plugin to their project.
-     2.   IDE have a poor support for Macro annotation, so it is likely your IDE will not know about the generated `Lens` (but it will compile). If you want a better IDE support, please vote on the following [issue](http://youtrack.jetbrains.com/issue/SCL-7419). 
-     3.   this solution can only be applied when you control the case classes since you need to annotate them. This means you cannot use this technique for classes defined in another project.
+     2.   poor IDE supports, at the moment only IntelliJ recognises the generated `Lens`.
+     3.   requires to have access to the case classes since you need to annotate them.
```
Comment (refried) [inline README.md]: Requires access to
or
Requires having access to

File: README.md [hunk 6]
```diff
@@ -115,40 +115,40 @@ There are 3 ways to create `Lens`, each with their pro and cons:
...
+     // or
+     val lenser = Lenser[Employee]
      val (_company, _name) = (lenser(_.company) , lenser(_.name))
      ```
 
-3.   Finally, the boiler plate free solution with macro annotation (which are probably the most experimental part of macros).
-     Adding `@Lenses` annotation on case classes will generate `Lens` for every single accessor of the case class.
-     These generated `Lens` are in the companion object of the case class (even if there is no companion object declared).
-     Nevertheless, this solution has several disadvantages: 
+3.   Finally, the fully automatic method using `@Lenses` macro annotation.
```
Comment (refried) [inline README.md]: using the `@Lenses` macro annotation

**Comment by julien-truffaut** [comment]
Thanks @NightRa  and @refried

**Pull request merged**